CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 1.2.0-beta.1 (Unreleased)
 
+### Breaking Changes
+
```

Comment:
Can you remove this section for now, please. I strongly suspect we won't need it for beta.1, esp since we don't do breaking changes between GA releases.

```suggestion
```